### DISCUSSION TITLE
Potrait Support

### DISCUSSION THREAD

**Discussion by trahkce** [phaselden/FlipIt#18] [Ideas]
Hi Flipit Dev,
Old Fliqlo support potrait mod, it would be nice if Flipit also do this, Thanks you ^^
![IMG_1702](https://user-images.githubusercontent.com/20172343/109615459-81d48000-7b66-11eb-82cf-1beeb364a781.jpeg)

**Comment by phaselden** [comment]
Interesting idea. I hadn't thought of doing that. Thanks. I will create a new issue for this.

**Reply by trahkce** [reply]
thanks you, big appreciate, love your works.

**Comment by thathman** [comment]
Looks like you forgot something...
